projects/components/src/table/table.component.ts

```diff
@@ -496,7 +496,7 @@ export class TableComponent
   }
 
   public isExpanderColumn(columnConfig: TableColumnConfig): boolean {
-    return columnConfig === this.expandableToggleColumnConfig;
+    return columnConfig.display === this.expandableToggleColumnConfig.display;
```

Comment:
So easy. Should there be any test to capture this particular failure?

Reply:
Agreed. Added.